client/sites-dashboard/components/sites-ellipsis-menu.tsx

```diff
@@ -246,102 +257,84 @@ const SiteDropdownMenu = styled( DropdownMenu )( {
-	isLaunched: boolean;
-} ) {
+function useSubmenuItems( site: SiteExcerptData ) {
 	const { __ } = useI18n();
-	return useMemo(
-		() =>
-			[
-				{
-					condition: isAtomic,
-					label: __( 'Database access' ),
-					href: `/hosting-config/${ siteSlug }#database-access`,
-					eventName: 'calypso_sites_dashboard_site_action_submenu_database_access_click',
-				},
-				{
-					condition: isAtomic,
-					label: __( 'SFTP/SSH credentials' ),
-					href: `/hosting-config/${ siteSlug }#sftp-credentials`,
-					eventName: 'calypso_sites_dashboard_site_action_submenu_sftp_credentials_click',
-				},
-				{
-					condition: isAtomic,
-					label: __( 'Web server settings' ),
-					href: `/hosting-config/${ siteSlug }#web-server-settings`,
-					eventName: 'calypso_sites_dashboard_site_action_submenu_web_server_settings_click',
-				},
-				{
-					label: __( 'Performance settings' ),
-					href: `/settings/performance/${ siteSlug }`,
-					eventName: 'calypso_sites_dashboard_site_action_submenu_performance_settings_click',
-				},
-				{
-					condition: isCustomDomain,
-					label: __( 'DNS records' ),
-					href: `/domains/manage/${ siteSlug }/dns/${ siteSlug }`,
-					eventName: 'calypso_sites_dashboard_site_action_submenu_dns_records_click',
-				},
-				{
-					condition: isAtomic,
-					label: __( 'GitHub connection' ),
-					href: `/hosting-config/${ siteSlug }#connect-github`,
-					eventName: 'calypso_sites_dashboard_site_action_submenu_connect_github_click',
-				},
-				{
-					condition: isAtomic,
-					label: __( 'Clear cache' ),
-					href: `/hosting-config/${ siteSlug }#cache`,
-					eventName: 'calypso_sites_dashboard_site_action_submenu_cache_click',
-				},
-				{
-					condition: isLaunched,
-					label: __( 'Privacy settings' ),
-					href: `/settings/general/${ siteSlug }#site-privacy-settings`,
-					eventName: 'calypso_sites_dashboard_site_action_submenu_privacy_settings_click',
-				},
-			].filter( ( { condition } ) => condition ?? true ),
-		[ __, isAtomic, isCustomDomain, isLaunched, siteSlug ]
-	);
+	const siteSlug = site.slug;
+
+	return useMemo< { label: string; href: string; eventName: string }[] >( () => {
+		return [
+			{
+				label: __( 'SFTP/SSH credentials' ),
+				href: `/hosting-config/${ siteSlug }#sftp-credentials`,
+				eventName: 'calypso_sites_dashboard_site_action_submenu_sftp_credentials_click',
+			},
+			{
+				label: __( 'Database access' ),
+				href: `/hosting-config/${ siteSlug }#database-access`,
+				eventName: 'calypso_sites_dashboard_site_action_submenu_database_access_click',
+			},
+			{
+				label: __( 'Github connection' ),
+				href: `/hosting-config/${ siteSlug }#connect-github`,
+				eventName: 'calypso_sites_dashboard_site_action_submenu_connect_github_click',
+			},
+			{
+				label: __( 'Web server settings' ),
+				href: `/hosting-config/${ siteSlug }#web-server-settings`,
+				eventName: 'calypso_sites_dashboard_site_action_submenu_web_server_settings_click',
+			},
```

Comment:
Should we add 'Web server logs' here?

Reply:
Done https://github.com/Automattic/wp-calypso/pull/73824/commits/9c6c56d741cf6cd3ccfa20f0bb5515eabc8a61b3